### ISSUE TITLE
DateCreated is null

### ISSUE THREAD

**Issue by Adnaram** [twilio/twilio-java#303]
Hi,

com.twilio.rest.api.v2010.account.Message.getDateUpdated function return a null value everytimes. I have the same problem for getDateCreated() and getDateUpdated(). 

Thanks,

This is my code : 
```
import com.twilio.Twilio;
import com.twilio.base.ResourceSet;
import com.twilio.rest.api.v2010.account.Message;
import com.twilio.type.PhoneNumber;


public static void main(String[] args)
{
	Twilio.init("...", "..."); 
	ResourceSet<Message> messages = Message.reader();
	//.setTo(new PhoneNumber("..."))
	//.setDateSent(DateTime.parse("2016-11-18")).read()

	// Loop over smss and print out a property for each one.
	for (Message sms : messages)
	{
		System.out.println(sms);        
	}
}
```

This is the answer i get : 

`Message{accountSid=..., apiVersion=2010-04-01, body=..., dateCreated=null, dateUpdated=null, dateSent=null, direction=inbound, errorCode=null, errorMessage=null, from=+..., numMedia=0, numSegments=1, price=-0.00750, priceUnit=USD, sid=..., status=received, subresourceUris={media=/2010-04-01/Accounts/.../Messages/.../Media.json}, to=+..., uri=/2010-04-01/Accounts/.../Messages/....json}`

**Comment by jingming** [comment]
@Adnaram what version of the library are you using?

**Comment by Adnaram** [comment]
I'm using "twilio-7.2.0-jar-with-dependencies.jar".

Thanks,

**Comment by jingming** [comment]
@Adnaram,

The code below works for me:
```java
Iterable<Message> messages = Message.reader().read();
for (Message m : messages) {
    System.out.println(m);
}
```

Outputs:
```
Message{accountSid=..., apiVersion=2010-04-01, body=..., dateCreated=2016-11-21T21:49:59.000Z, dateUpdated=2016-11-21T21:50:00.000Z, dateSent=2016-11-21T21:49:59.000Z, direction=outbound-api, errorCode=null, errorMessage=null, from=..., messagingServiceSid=..., numMedia=0, numSegments=1, price=-0.00750, priceUnit=USD, sid=..., status=delivered, subresourceUris=..., to=..., uri=...}
```

Does this fail for you on all messages?

**Comment by Adnaram** [comment]
Yes. I get  `dateCreated=null, dateUpdated=null, dateSent=null` for every SMS no matter it is oubound-api or inbound.

**Comment by efossier** [comment]
@Adnaram I tried your exact code snippet (with my creds) and was able to get non-null dates. Can you answer a couple of questions for me to see if we can narrow this down a bit?

- What version of the `joda-time` dependency do you have installed?
- Is this happening on every sms you have ever sent? Or just for recently sent ones? Dates are populated on messages after the message has been sent so if you are trying to read a newly sent message its normal that the dates be null for some short amount of time.

**Comment by Adnaram** [comment]
@efossier

I'm using java 1.7. I'm using [twilio-7.3.0-jar-with-dependencies.jar](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.twilio.sdk%22) so I dont have to import any dependency. Jodo-time is included in the jar file. 

This is happenning on every SMS since I started using Twilio. I tried to read the message's dateCreated from different days but I always have this problem. I just tried to read the yesterday's SMS and it didn't work.

Thanks,

**Comment by efossier** [comment]
@Adnaram I tried again with that exact jar and still seeing non-null dates... Can you elaborate on what you are using to build your project (maven, command line)? Are you using any other third party dependencies that you are linking in?

**Comment by Adnaram** [comment]
I found the problem. 

The problem is in `com.twilio.converter.DateConverter `

My local is `ca_FR` and the rfc2822DateTimeFromString() function works only with `en_US` locale.

Is it possible to do a fix for this in the API ?

```
DateTimeFormatter formatter = DateTimeFormat.forPattern(RFC2822_DATE_TIME_FORMATTER);
DateTime dateTime = formatter.withLocale(new Locale("en_US")).parseDateTime(...);
```

Thanks,

**Comment by efossier** [comment]
@Adnaram Good catch, I've created a PR to fix it.

**Comment by efossier** [comment]
@Adnaram fix is included in latest release `7.4.0` https://search.maven.org/#artifactdetails%7Ccom.twilio.sdk%7Ctwilio%7C7.4.0%7Cjar
